package.json

```diff
@@ -13,6 +13,14 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
+    "babel-polyfill": "^6.26.0",
```

Comment:
what is babel for? I don't see it used anywhere. If it's used and it's an important dependency, let's add an entry about it in the README. If it's not used it should not be in master. Same goes with chai, there shouldn't be any code that it's not used in master

Reply:
Yea babel is a dependency for the use of async/await in tests, Chai also a library I'm using in `test/LiquidDemocracy.test.js`. Agree we can make a note of this in the README, we can do that in a future PR maybe when addressing #14